### PR TITLE
feature: make responder this.handle callback to a method

### PR DESCRIPTION
**Pull request by sydcanem**
- This will make the method patch-able for tracing purposes.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
     "jshint": "jshint",
     "eslint": "eslint",
     "jscs": "jscs",
-    "nsp" : "nsp"
+    "nsp" : "nsp",
+    "lint": "npm run eslint -- ./ && npm run jscs -- ./"
```
Comment (openam) [inline package.json]: Does the older version of eslint need the `--reset` stuff in it?

**Comment by openam** [comment]
Looks good to me other than if the  eslint --reset is needed.

File: package.json (hunk 1 above)
Comment (sydcanem) [inline package.json]: I added --reset and I didn't any difference on the results.

**Comment by yamii** [comment]
I'm good with this :dancers:

**Pull request closed without merging**